internal/condition/deployment_config_condition.go

```diff
@@ -0,0 +1,35 @@
+package condition
+
+import (
+	"errors"
+	"fmt"
+	"github.com/fabric8-services/fabric8-jenkins-idler/internal/model"
+	"time"
+)
+
+// DCCondition covers changes to Jenkins DeploymentConfigs
+type DeploymentConfigCondition struct {
+	Condition
```

Comment:
`Condition` is referenced here, but not in other conditions, is that intentional?

Reply:
ahh, my wrong. I thought I removed it. I pushed an update ...